templates/cluster-template.yaml

```diff
@@ -1,6 +1,8 @@
 apiVersion: cluster.x-k8s.io/v1alpha3
 kind: Cluster
 metadata:
+  labels:
+    cni: calico
```

Comment:
@sedefsavas can you please confirm that adding this to the released cluster template won't have any negative effect on users who use it without cluster resource sets and continue to apply the CNI addon manually?

Reply:
Yes, as long as there is no `ClusterResourceSet` object that has this label as selector, it will be a no-op.
`ClusterResourceSet` labels are user selected, so there is no special meaning of any of them.